modules/core/arrow-core/src/main/kotlin/arrow/core/extensions/sequenceK.kt

```diff
@@ -317,6 +318,11 @@ interface SequenceKSemialign : Semialign<ForSequenceK>, SequenceKFunctor {
     }.k()
 }
 
+fun <A, B, C> SequenceK<A>.alignSequenceWith(other: SequenceK<B>, fab: (Ior<A, B>) -> C): SequenceK<C> =
+  SequenceK.semialign().run {
+    alignWith(this@alignSequenceWith, other, fab)
+  }.fix()
+
```

Comment:
What is this needed for? Is this just an specialisation for the semialign alignWith?

Reply:
i have removed it